tests/TestCase.php

```diff
@@ -17,22 +18,27 @@ abstract class TestCase extends OrchestraTestCase
 {
     use RefreshDatabase, WithWorkbench;
 
+    private Core $core;
```

Comment:
I'm capturing the object here, as one of the tests manipulates the core class in the container and then it fails to resolve it again in the `tearDown` method.

This makes it more bulletproof.